engine/crates/parser-sdl/src/rules/federation/field_set.rs

```diff
@@ -0,0 +1,143 @@
+use std::str::FromStr;
+
+use engine::registry::{self, field_set::Selection};
+use nom::{
```

Comment:
nice to know that we have nom already in there to reach out when we need it

Reply:
Yeah.  I think we have `nom`, `combine` _and_ `pest` used in various bits of our codebase.  Possibly excessive having all three